<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="87645505-55ca-4cbe-b06d-c04082160499" name="dabed@MIC5925 2010-06-15 10:26:40" runUser="RUSSELL\dabed" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2006">
  <TestRunConfiguration name="Default Run Configuration" id="04214ea7-69a1-4a99-b3da-68a0cf8e7b71">
    <Deployment runDeploymentRoot="D:\perso\PortableGit-1.7.0.2\MatozDz\TestResults\dabed_MIC5925 2010-06-15 10_26_40">
      <DeploymentItem filename="D:\perso\PortableGit-1.7.0.2\MatozDz\MatozDz\bin\MatozDz.dll" />
      <DeploymentItem filename="D:\perso\PortableGit-1.7.0.2\MatozDz\MatozDz.Tests\Lib\Moq.dll" />
    </Deployment>
    <Hosts skipUnhostableTests="false" />
    <TestTypeSpecific />
  </TestRunConfiguration>
  <ResultSummary outcome="Error">
    <Counters total="1" executed="1" error="0" failed="0" timeout="0" aborted="1" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <RunInfos>
      <RunInfo computerName="MIC5925" outcome="Error" timestamp="2010-06-15T10:27:02.3807174-04:00">
        <Text>Test host process exited unexpectedly.</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
  <Times creation="2010-06-15T10:26:40.1304048-04:00" queuing="2010-06-15T10:26:41.4282094-04:00" start="2010-06-15T10:26:41.5845714-04:00" finish="2010-06-15T10:27:02.3807174-04:00" />
  <TestDefinitions>
    <UnitTest name="Detail_Can_Handle_invalid_value" storage="d:\perso\portablegit-1.7.0.2\matozdz\matozdz.tests\bin\debug\matozdz.tests.dll" id="0eae54b2-9f54-aa5c-b20b-9a781f2d57d7">
      <Css projectStructure="" iteration="" />
      <Owners>
        <Owner name="" />
      </Owners>
      <Execution id="cae5a3e0-bac5-4873-b372-6e3439902657" />
      <TestMethod codeBase="D:\perso\PortableGit-1.7.0.2\MatozDz\MatozDz.Tests\bin\Debug\MatozDz.Tests.dll" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="MatozDz.Tests.Controllers.MagasinsControllerTest, MatozDz.Tests" name="Detail_Can_Handle_invalid_value" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="0eae54b2-9f54-aa5c-b20b-9a781f2d57d7" executionId="d17f1fa9-549c-4503-a717-a360e98a61c1" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="d17f1fa9-549c-4503-a717-a360e98a61c1" testId="0eae54b2-9f54-aa5c-b20b-9a781f2d57d7" testName="Detail_Can_Handle_invalid_value" computerName="MIC5925" duration="00:00:20.6397840" startTime="2010-06-15T10:26:41.7409334-04:00" endTime="2010-06-15T10:27:02.3807174-04:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Aborted" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d">
    </UnitTestResult>
  </Results>
</TestRun>